### PR TITLE
Fix typo: Update design_in_detail.md

### PR DESCRIPTION
**Pull request by c4pt0r**
Fix typo

### CODE REVIEW

File: docs/design_in_detail.md [hunk 1]
```diff
@@ -170,4 +170,4 @@ The built-in TTL mechanism of KubeBrain exists as a component of the "Compact" m
...
 
-First of all, binlog is not a feature that all distributed kv databases must have. For distributed kv databases, enabling binlog will greatly increase write latency and reduces system throughput. In the K8s system, the event generated when the metadata is changed is used to incrementally update the cache. When the KubeBrain node crashes and the memory event is lost, the cache can be resynchronized through the list watch through the API Server, which can tolerate some event loss. Of course, subsequent projects will continue to optimize in this regard.
\ No newline at end of file
+First of all, binlog is not a feature that all distributed kv databases must have. For distributed kv databases, enabling binlog will greatly increase write latency and reduces system throughput. In the K8s system, the event generated when the metadata is changed is used to incrementally update the cache. When the KubeBrain node crashes and the memory event is lost, the cache can be resynchronized through the list watch through the API Server, which can tolerate some event loss. Of course, subsequent projects will continue to optimize in this regard.
```
Comment (divanodestiny) [inline docs/design_in_detail.md:173]: Sorry, i can't find the changes in this line. Can you explain the changes?

**Comment by divanodestiny** [comment]
@c4pt0r Thx for your PR~ Would you mind to format the commit msg? like  `docs(design_in_detail): fix typo`

File: docs/design_in_detail.md (hunk 1 above)
Comment (STRRL) [reply docs/design_in_detail.md:173]: It appends an extra `\n` at the end of the line. :P

image

ref: https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

Comment (divanodestiny) [reply docs/design_in_detail.md:173]: Got It.

**Comment by c4pt0r** [comment]
Sure, thank you for your reply, and sorry for the late response. I will describe it in more detail

**Comment by c4pt0r** [comment]
PTAL, it's a small fix though :)

**Review by divanodestiny — APPROVED** [review]

**Pull request merged**